index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>VL Survey Form</title>
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+  <div class="container">
+    <header>
+      <h1 id="title">VL Survey Form</h1>
+      <p id="description">
+        Thank you for taking your time to improve the platform
+      </p>
+    </header>
+    <form id="survey-form">
+      <!--* Name -->
```

Comment:
Plsease do not leave this notes in your codebase. It can confuse beginners programmers